client/src/routes/Profile/profile.module.css

```diff
@@ -65,14 +63,15 @@
 
 /* 360px */
 @media only screen and (min-width: 22.5rem) {
-    .profileContainer {
-        margin: var(--spacer-4) 0;
-    }
 
     .img {
         width: 60px;
         height: 60px;
     }
+
+    .profileContainer {
+        margin-bottom: var(--spacer-3);
+    }
     
     .name {
         font-size: var(--h4mobile);
```

Comment:
No font-sizes should appear in media queries.  They will be controlled globally.